python-sdk/langwatch/README.md

```
@@ -0,0 +1,113 @@
+Please make sure to update tests as appropriate and follow our coding standards.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+
+## Support
+
+- Documentation: [docs.langwatch.ai](https://docs.langwatch.ai)
+- Issues: [GitHub Issues](https://github.com/langwatch/langwatch-saas/issues)
+- Discord: [Join our community](https://discord.gg/langwatch)
+- Email: support@langwatch.ai
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)